TWNIA2/1. User Guides/4. 套件與軟體.md

```diff
@@ -1 +1,59 @@
+---
+title: 套件與軟體 | en
+tags: Guide, TWNIA2, EN
+GA: UA-155999456-1
+---
+
+{%hackmd @docsharedstyle/default %}
+{%hackmd @docsharedstyle/twccheader-en %}
+
+# Packages and software
+
+The packages and software available for Taiwania2 (HPC CLI) are as follows:
+
+
+**【Software tools and Libraries】**
+
+- Mathematics software
+
+| Software name | Software version | 	Software platform |
+| -------- | -------- | -------- |
+| [Matlab](https://iservice.nchc.org.tw/nchc_service/nchc_service_software_detail.php?diuu=c13edee6-ed57-4fed-9be1-5480b4a05302)       | For educational use     | UNIX, PC    |
+
+**【Chemical and biological software and databases】**
+ 
+ - Computational chemistry software
+
+| Software name | Software version | 	Software platform |
+| -------- | -------- | -------- |
+|  [LAMMPS](https://iservice.nchc.org.tw/nchc_service/nchc_service_software_detail.php?diuu=a0c87aa8-28aa-4155-ac7f-b140cf0de908)         | For educational use, industrial edition     | UNIX     |
+|  [Gaussian 16 (Taiwania 2)](https://iservice.nchc.org.tw/nchc_service/nchc_service_software_detail.php?diuu=3f5c7a44-64c4-460b-8117-b1a4354e575e)          | For educational use     | UNIX     |
+
+ - Biomedical software
+
+| Software name | Software version | 	Software platform |
+| -------- | -------- | -------- |
+|  [GROMACS](https://iservice.nchc.org.tw/nchc_service/nchc_service_software_detail.php?diuu=6e03663f-b562-4bc1-8476-5e0dca7c574a)           | For educational us     | UNIX     |
+|  [NAMD](https://iservice.nchc.org.tw/nchc_service/nchc_service_software_detail.php?diuu=17d3c4f5-e655-4143-ae6c-76d6d453b6d8)             | For educational use     | UNIX     |
+|  [AMBER](https://iservice.nchc.org.tw/nchc_service/nchc_service_software_detail.php?diuu=867baa76-007b-485b-9ebd-63560d488cb8)               | For educational use     | UNIX     |
+
+
+**【AI frameworks】**
+
+- AI related software
+
+| Software name | Software version | 	
+| -------- | -------- | 
+|  [TensorFlow](https://iservice.nchc.org.tw/nchc_service/nchc_service_software_detail.php?diuu=717a850d-4fb6-4033-a2f8-cd0d52cdf5b8)            | OpenSource edition     |     
+|  [PyTorch](https://iservice.nchc.org.tw/nchc_service/nchc_service_software_detail.php?diuu=66a576cb-56e9-4185-a328-a2ab806b38cf)               | OpenSource edition     |    
+
+
+:::info
+{%hackmd @docsharedstyle/note-en %}
+- How to use AI framework software? Please refer to [<ins>Create TWNIA2 container</ins>](https://man.twcc.ai/@twccdocs/howto-twnia2-create-sglrt-container-en) and [<ins>Multi-nodes parallel computing of container</ins>](https://man.twcc.ai/@twccdocs/howto-twnia2-run-parallel-job-container-en).
```

Comment:
container > containers

Reply:
Multi-nodes parallel computing of container > Run parallel computing jobs with multi-nodes on containers